hack/lib/util.sh

```diff
@@ -157,4 +164,37 @@ kube::util::gen-doc() {
   rm -rf "${tmpdir}"
 }
 
+# Takes a path $1 to traverse for md files to append the ga-beacon tracking
+# link to, if needed. If $2 is set, just print files that are missing
+# the link.
+kube::util::gen-analytics() {
+  local path="$1"
+  local dryrun="${2:-}"
+  local mdfiles link
+  # We don't touch files in Godeps|third_party, and the kubectl
+  # docs are autogenerated by gendocs.
+  mdfiles=($( find "$1" -regex ".*\.md" \
+              -type f -not -path "*Godeps/*" \
```

Comment:
This will match `bogusGodeps` no? Why the `*` at the beginning? Also, I think `-name` might work here, perhaps together with `-prune`.

Maybe take a look at this?
http://stackoverflow.com/a/4210072